src/CI/MissingFilesChecker.php

```diff
@@ -32,13 +33,8 @@ public function getMissingFiles(): array
         $orphanedFiles = [];
 
         foreach ($finder as $file) {
-            $sourcePath = ltrim(substr($file->getPathname(), \strlen($this->buildContext->getSourceDir())), '/');
-
-            $htmlFile = sprintf(
-                '%s/%s.html',
-                $this->buildContext->getOutputDir(),
-                substr($sourcePath, 0, -4)
-            );
+            $sourcePath = u($file->getPathname())->after($this->buildContext->getSourceDir())->trimEnd('/');
```

Comment:
because we're doing `ltrim` shouldn't we use `trimStart`?

Reply:
You are right! Fixed. Thanks!